roles/teuthology_nameserver/tasks/dnssec-keygen.yml

```diff
@@ -0,0 +1,44 @@
+#Temporarily moved dnssec key generation from deploy-teuthology script to the ansible task due to HMAC-MD5 being depricated on Tumbleweed.
+- name: Generate new keys
+  shell: "/usr/sbin/dnssec-keygen -r /dev/urandom -a HMAC-MD5 -b  512 -n HOST '{{ zone_name }}'"
+  register:
+    dns_keygen
+  tags:
+    - dnssec_keygen
```

Comment:
Since the tags are used in outside import_tasks then dnssec_keygen tag is redundant for each task of this file.